charms/mlflow-server/tests/unit/test_operator.py

```diff
@@ -373,3 +373,80 @@ def test_install_with_all_inputs(harness, mocker):
         mlflow_pod_defaults_minio_data["MLFLOW_TRACKING_URI"]
         == f"http://{harness.model.app.name}.{harness.model.name}.svc.cluster.local:{harness.charm.config['mlflow_port']}"  # noqa: E501
     )
+
+
+def test_install_without_nodeport_and_lb_services(harness, mocker):
+    harness.set_leader(True)
```

Comment:
Please add doc string indicating what is being tested, eg. `"""Test deployment without LB services."""`